governance/votes/Executive vote - March 29, 2023.md

```diff
@@ -0,0 +1,94 @@
+- The RETH-A [Debt Floor (`dust`)](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-floor) will be decreased, as detailed below.
+- Monetalis updates:
+	- Excess funds declaration, as detailed below.
+	- Removal of DC-IAM from RWA-007, as detailed below.
+	- Increase of the [MIP65](https://mips.makerdao.com/mips/details/MIP65) [Debt Ceiling (`line`)](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling), as detailed below.
+- PSM Parameter Normalization, as detailed below.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. This is to ensure that at least some auction keepers are available to react to unforeseen issues with their bots in the event of early liquidations on new collateral types.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Recognized Delegate Compensation
+
+As per [MIP61: Recognized Delegate Compensation](https://mips.makerdao.com/mips/details/MIP61), a total of **3,307 DAI** will be distributed to two Recognized Delegates as detailed below, if this executive proposal passes. _Note that the compensation for these two Recognized Delegates for February was left uncleared as they [failed to confirm their addresses](https://forum.makerdao.com/t/recognized-delegate-compensation-february-2023/20033/2). They have now been confirmed_:
+
+- **3,126 DAI** will be transfered to [London Business School Blockchain](https://vote.makerdao.com/address/0xf1792852bf860b4ef84a2869df1550bc80ec0ab7) at [`0xB83b3e9C8E3393889Afb272D354A7a3Bd1Fbcf5C`](https://etherscan.io/address/0xB83b3e9C8E3393889Afb272D354A7a3Bd1Fbcf5C).
+- **181 DAI** will be transfered to [ConsenSys](https://vote.makerdao.com/address/0x40f784b16b2d405efd4e9eb7d0663398d7d886fb) at [`0xE78658A8acfE982Fde841abb008e57e6545e38b3`](https://etherscan.io/address/0xE78658A8acfE982Fde841abb008e57e6545e38b3).
+
+### MKR Transfers
+
+As per the ratified [MIP40c3-SP17: Sustainable Ecosystem Scaling Core Unit MKR Budget](https://vote.makerdao.com/polling/QmSmhV7z), **229.78 MKR** will be transferred to Sustainable Ecosystem Scaling Core Unit as detailed below, if this executive proposal passes:
```

Comment:
Not sure if the CU name needs to be repeated in the paragraph and the bullet point. Also consider adding link to CU - https://mips.makerdao.com/mips/details/MIP39c2SP10, and formatting consistency "(SES-001)" absent here but present below. "(CES-001)" present in both instances below. Sorry for being picky!

Reply:
Love it actually :)